features/support/hooks.rb

```diff
@@ -1,4 +1,7 @@
 Before do
+  # Reset the current user:
+  User.current_user = nil
```

Comment:
The issue that I ran into was creating works/series/chapters with `FactoryGirl`. It would treat all of the creatorships as if `User.current_user` was inviting new co-creators, and as a result either (a) the creatorship wouldn't be saved because it was "inviting" a user who had co-creation disabled, or (b) the creatorship would be treated as an invitation. In either case, the work/series/chapters would have no approved creators, and would fail to save.

There might be a way to handle the issue more consistently by temporarily overriding `User.current_user` in the factory, but I'm not sure how to do that.

Reply:
Setting it to nil in a before hook is fine, we already do the same for specs. We should have done this anyway, even without the new creatorship specs breaking.